inst/staticexports/ng_fpr.R

```diff
@@ -165,3 +165,89 @@ sfpr_create_hydrograph <- function(
 }
 
 
+#' Determine replacement structure type and size based on measured field metrics.
+#' @param dat PSCIS data
+#' @param fill_dpth standard fill depth, default is 3m.
+#' @param brdg_wdth standard bridge width, default is 15m.
+#' @param chn_wdth_max maximum channel width where the bridge should start to be more than brdg_wdth, default is brdg_wdth - 5m.
+#' @param fill_dpth_mult for every 1 m deeper than 3m, we need a 1.5:1 slope so there is 3m more bridge required
+#'
+#' @importFrom dplyr mutate filter select case_when
+#' @importFrom plyr round_any
+#' @importFrom readr write_csv
+#' @importFrom chk chk_numeric
+#'
+#' @export
+#'
+#' #' @examples \dontrun{
+#' fpr_structure_size_type(dat)
+#' }
+#'
+sfpr_structure_size_type <- function(
+    dat = NULL,
+    fill_dpth = 3,
+    brdg_wdth = 15,
+    chn_wdth_max = brdg_wdth - 5,
```

Comment:
including a param in a param is not something i have seen done before i odn't think. ill prob change and put that calc in the code of the funciton

Reply:
okay sure. Probably safer that way.